pyserini/trectools/_base.py

```diff
@@ -0,0 +1,99 @@
+#
+# Pyserini: Python interface to the Anserini IR toolkit built on Lucene
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import pandas as pd
+from typing import List
+
+
+class TrecRun:
+    """Wrapper class for a trec run.
+
+    Parameters
+    ----------
+    filepath : str
+        File path of a given Trec Run.
+    """
+
+    def __init__(self, filepath: str = None):
+        self.run_data = dict()
+        self.filepath = filepath
+
+        if filepath is not None:
+            self.read_run(self.filepath)
+
+    def read_run(self, filepath: str) -> None:
+        run_data = pd.read_csv(filepath, sep="\s+", names=["topic", "q0", "docid", "rank", "score", "tag"])
```

Comment:
changed to single quotes? we've standardized rest of code base on single quotes.

Reply:
done